cookbooks/gathering_contexts_when_using_multiple_contexts.rst

```diff
@@ -1,11 +1,12 @@
-Accessing contexts from each other
-==================================
+Gathering Contexts when using Multiple Contexts
```

Comment:
the previous title seems better IMO. What people want to do is accessing contexts from each other. Gathering them is the way to solve the issue, but it is not what people will be looking for (if they know it is what they need, they won't need to read the article as they already know how to do things)